requirements-devel.txt

```diff
@@ -1,2 +1,3 @@
 # requirements for a development environment
 -e .[devel]
+scriv
```

Comment:
this inspired me for https://github.com/datalad/datalad-extension-template/pull/56 . Then here it would be `-e .[devel,devel-utils]` if so really needed. But since this has nothing to do with adding metadata extractor -- shouldn't even be in this PR. Feel welcome to submit a separate PR with such changes.